### PR TITLE
Add support for running tests on Windows

### PR DESCRIPTION
**Pull request by steven-supersolid**
cross-env or similar is required for setting environment variables
js paths instead of cmd paths required on Windows

I'm not sure if we need ./node_modules/.bin/babel-node as babel-istanbul includes babel? Would like to remove if not required to simplify and slim the package down.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,14 +32,15 @@
...
     "deep-diff": "^0.3.3",
     "jasmine": "^2.3.2",
     "mongodb-runner": "^3.1.15"
   },
   "scripts": {
     "build": "./node_modules/.bin/babel src/ -d lib/",
-    "pretest": "MONGODB_VERSION=${MONGODB_VERSION:=3.0.8} mongodb-runner start",
-    "test": "NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node ./node_modules/.bin/babel-istanbul cover -x **/spec/** ./node_modules/.bin/jasmine",
+    "pretest": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=3.0.8} ./node_modules/.bin/mongodb-runner start",
+    "test": "cross-env NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node ./node_modules/babel-istanbul/lib/cli.js cover -x **/spec/** node_modules/jasmine/bin/jasmine.js",
```
Comment (steven-supersolid) [inline package.json]: I release I dropped a ./ on the path to jasmine. Not sure what the best practice is here but would change to be consistent

**Comment by drew-gross** [comment]
Cool! @steven-supersolid If you can also update our `.travis.yml` to build on windows that will help us make sure windows test keep working into the future.

**Comment by steven-supersolid** [comment]
@drew-gross That's still something I have to learn, but preliminary look shows Windows might not be supported by Travis? https://github.com/travis-ci/travis-ci/issues/216

**Comment by drew-gross** [comment]
Ah. I saw that issue, didn't want to read the whole thing, and assumed since it was closed, and I knew OSX was now supported, that meant windows was also supported, My bad!

**Pull request merged**